src/server/req_manager.c

```diff
@@ -1561,17 +1612,27 @@ mgr_sched_set(struct batch_request *preq)
 	int	  bad_attr = 0;
 	svrattrl *plist;
 	int	  rc;
+	pbs_sched *psched;
+
+	psched = find_scheduler(preq->rq_ind.rq_manager.rq_objname);
+	if (!psched) {
+		req_reject(PBSE_UNKSCHED, 0, preq);
+		return;
+	}
+	if (validate_sched_attr_vals(preq, psched))
```

Comment:
I do not understand how it is an overhead? Isn't it the whole purpose of having attribute action functions. Even in this function you are validating a bunch of attributes. An action function can always return an error number which is then given back to the caller as a reply.